packages/a-msgpack/src/Encoder.ts

```diff
@@ -359,6 +359,11 @@ export class Encoder {
   }
 
   encodeExtension(ext: ExtData): void {
```

Comment:
looks like you don't need to split this function anymore. it's up to you if you want to revert it